dev/BBimie/Wine quality.ipynb

```diff
@@ -0,0 +1,1828 @@
Rq7xG6WV7bn7nQSTveoQWYfDt9OJn6+sDd6fvhUZytnGuKxCdgtMJg72lxunydN+WITw7jqNGV1ZC6D0KfCe37SZaMoiemp6RTEg+SYuKI4t8LyQvfoOu9VjCFjCDMI6vTqgHrwC+0M5jVT6PAwlvtZMJ9dOk1PjfRQjTwvNTkVPxpnf/zlwbszjh6t1hD7N216fZN7wzLERA1cbp4XuU6HXOTi98YQm0UoOfZQrdjRBAH9DiBbIULblqWvUOIoTUyYRKaEI61oR0rCvSA79mUeeQ6nApOR/uit+6E66qoyvrntunVzrG/YQr4DjCfW7D1CDuSeiIm5VKOm/MPhj4Bam3nX67KgmEFetUVtZ73YHoHPwg9/sthFvolYR65A/EqCBTA26UGrWmGNHrdL1XT/d7PcIl98epsexDdNCuocqRTsW7chQhtI8mRlwvkjICpHdtHRqQqZaw472anpssceVXCJVwQ7K2zlOnZt/0Jj5s3SrWexNGtiyZ2xWEK+DAAuvQhxiqbgo8nbadSQQGtRn6TxgOsxD0MakxnkDYClYjPFhqznNf5blMSY13tQFulb20RdKL8aMkPI4iVC8/JrxkCk9/UUWdDyNiOPYhZqmaTMvI4GZCZdOjTmUdRXjIHER0Sn6Qtm9PpK1+L5U5Oa2fSPRsnwF2bfa1aue5LpOE2qJED/0uolOQjQiHpmswlhilt/udJDLz/oKwqf2VlujhWcDXCj6/vDAaTGgdViaM0d8m2SgINe9NNKHDk/VySkGayH1Pdz8/rXfzXPi5mV1I6Nt/QEjtr7n7Ew2o16GET/+pZrYz8UJ/y2MC+fx+/6uvme1D2DKucfcfpG27EUPKJ4Fb3P2vBdU3m2h+dSKC86m0/RLC4H6mx6Te8/t/L3f/V/o+gnhRnk7r1xC9s71oEWh/cfc3iziXakih/70Ig/BUd7/fzAYRkcPdCR/018xsOXf/cx3KWpR4Bi/ySK2BmT1BNGB9iV7xe8S9HpD+uizhrvsbd/9pfqL1zo6ZZRPJPO/u3zSzAwiV5uXAL9z9b2nS+bfc/doOHD9zQ/0qMbrcmbiW/yZsL18m4lHq/r5UTHh/IOHZtbq772lmvQhHkJ2JczvXzD5V1Hu7QJrRA2jWQkjjD5nb+yAf0LER8cA8RcHpdJm7lzCO6M2fQW6+2op9lqbFO+jTaXmZNMlIbr+9CLVBv4LrPzZdpzsI43bm9XAh0ZNt1feZcL08lMgYeSThNvcL4Ie5faYSPe7eRZ5DB875LCIILvMsG05Es55CnbxMaFFBXUxuciBCbfEJofLbnBiFLJKu/c+Ag5t9fWo4557ESOdK4Li07WAiDck3CRvTH4BtaihjUSKJ24NpvRvwd6LT1acB57gHYfM4lMhXdlDa3oMYGXyDAt3S26xfsx+Chp1oS1DYpkT+lxNzv80Tvp4+6zWpTPZy98k/dLntixMqnXPmJ4DSi3JBaoheTNvWJ4bMh1fsW+iDTehXHyF6/xNT4zSFZIsg5Ueaz3/HJEFxDKHvzrxcnmDulM2XsABX04LPr1JtlbmE7kKof/ZK9zKbVKbDnjnze8ZSg/E44TXWjeg1nkcYiE8nGTSJwL3vU3CAYMHXe3D63CoJg2z2v30Jm93FtAQfdthZgOiN/5LojOxEqDJXbMD5jSLUzFmA6uaEYDs4e75YwGxwDbkHzX4IGnqy0Yv9LqH/f49cPm9ys/vUSwBUlL0Loff8IfDV3PZ5HmzmHgkMpyWnzj1EBs4tc79vQPK1b+B1XJnI/LkdMYJZn9Dt308ydLfyn/zI6wTC4Hcfc3sb/Qp4oMnPSP7a70JFDqPUQH+HMMjOpH6TymzJvHMDnEjEYdxGeF2tQui4z0nX7mtp+5b1qEOTrvcShB3onLSeCYPMiDshvbM719pYEqOCLxHZb2fRylzY9X6G0vrmRGzAbbQE+W1KjAwOaPY9cC+RICCCbx4n5aZJjdmbwJcaUHaWFmAwYQx+jrmjGOfXK+xO9IqWS0s2O9rUJCAyT5GRhJtdXTMStlKf9Ylheja6OoNIXw0RBHY3bRiLCYPrTYRO+HbCGJoPnnuABhm426jn5wnB25rX1hJE77IuHiapkZtGyzzGeaG5ciYAKsrfnkiPvEWzr1UHzjd/Lbun5/eHpAmJCKF4PS1G1K8Qo8yaR7qEOu1/81oUfG7rEqOPbsTI7ry0ZM/URjRgruOq6t3sCjTsRONmXJ/vBaRG9hNamRGrxrKWIXq9WTDMdull3yMJhKFp+3yH87S4EHYjYhmuIAW0EUPl7yXhcCQRZVp4JCShBvpZrh7jiSHukem8FujpRAiy35BUPoSa6EYicG9Yk5+P/Au8SRJSWZBP98p96lju54jR0W75MoooqzMtzB2b0z01mj+ixUawDTlPMQru5BRwfscQruH3pHdmBGEY/gbhKTSg2XXMLwvtxDTZJA9mtpKZLUM8bLMJYZDxKqF3n1Hn4lcheuzHpYlljIggnETYAF5NE2ucZWZLt1Z3b/FmWpmIHJ0NbG9m27n7sUT6hguJ4KU/uft/6nwO+foMBHD3qYTL4kQz24UY0p9NDHPP9rYnOVkBuNnDw6aHu/+E0G/vCGxkZj1amZyjIXh6e82sB9E56AWMNbNB7v7f/D71IpX1ARFZPSJ5U3kRZXUGcu/kMELlM83M1k3X93liRHmEmZ3s7g+4+ywz6w7g7u82q97txczWIdJ8bOXuOxJ2iS8TMT0/INTSnYtmS6KCpfKOhFT+GvBY2nYrcWO+TvRms+CtmntgRMTiAUTDvw1hWPxS+u1bhE55GBHA9jxtpKwgooXvIYzJfYlG/1KS9wThw1736NF03EPS92WJHmt+fuCJhHor6zFXOyfsjul8Vs9t25kY7RSeT6WK896EaIyyRGaXEuqXuqdpIHrE56V7uzVhc/gcTQqaa+A1zuY3XpaIlXieuQ3fF1LQvN8FnlOlTWA5osFfLbftduD09H2xZte5cllo4wjMbA2
```

Comment:
It looks like you are using the full dataset rather than the training set to train the model. This should be `train`